tempest/api/workloadmgr/base.py

```diff
@@ -3074,3 +3074,109 @@ def get_quota_type(self):
             resp.raise_for_status()
         quota_types = json.loads(resp.content)
         return quota_types['quota_types']
+
+    '''
+    This method returns the quota type id of the specified type
+    '''
+    def get_quota_type_id(self, quota_type):
+        resp, body = self.wlm_client.client.get(
+            "/project_quota_types")
+        LOG.debug("get_quota_type response: %s", resp.content)
+        if(resp.status_code != 200):
+            resp.raise_for_status()
+        quota_types = (json.loads(resp.content))['quota_types']
+        quota_type_id = None
+        for i in range(len(quota_types)):
```

Comment:
instead of range you can iterate over quota_types itself.

Reply:
Yes, will update in code